zipline/assets/__init__.py

```diff
@@ -20,6 +20,15 @@
     make_asset_array,
     CACHE_FILE_TEMPLATE
 )
+from .asset_writer import (
+    NullAssetDBWriterLegacy,
+    AssetDBWriterLegacyFromList,
+    AssetDBWriterLegacyFromDictionary,
+    AssetDBWriterLegacyFromDataFrame,
+    AssetDBWriterLegacyFromReadable,
+    FUTURE_TABLE_FIELDS,
+    EQUITY_TABLE_FIELDS
```

Comment:
Why do these FIELDS lists need to be imported? Are the fields referenced outside of the writer/finder?